tests/e2e/csi_snapshot_basic.go

```diff
@@ -1624,6 +1750,31 @@ var _ = ginkgo.Describe("Volume Snapshot Basic Test", func() {
 		snapshotCreated, snapshotContentCreated, err = deleteVolumeSnapshot(ctx, snapc, namespace,
 			volumeSnapshot, pandoraSyncWaitTime, volHandle, snapshotId, true)
 		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
+		if isStorageQuotaFSSEnabled && supervisorCluster {
```

Comment:
Please check if this can be seperated into 3 different functions:

1. For volume and snapshot quota and usage before
2. For volume and snapshot  quota and usage after
3. For cleanup of volumes and snapshots

Reply:
I will take it up in next PR